azure-pipelines/templates/create-github-release.yml

```diff
@@ -0,0 +1,34 @@
+parameters:
+  GitHubConnection: "" # defaults for any parameters that aren't specified
+  releaseNotesSource: inline
+  addChangeLog: true
+  isPreRelease: false
+  isDraft: false
```

Comment:
let's set this to true here, or pass it in as true

Reply:
I don't care either way but why?

Reply:
@mydiemho we may want to provide our own release notes aside from those rolled up via git commits

Reply:
> @mydiemho we may want to provide our own release notes aside from those rolled up via git commits

Sure, I name the file `CHANGELOG.md`, let me know if you want it to be name something else.  

The `RELEASE NOTES` section will be whatever you have in that file.  The `CHANGES` section is auto-generated by pulling all commits since last release.

<img width="1070" alt="Screen Shot 2019-10-15 at 21 51 48" src="https://user-images.githubusercontent.com/1634185/66889141-308a1780-ef96-11e9-984c-b00d7f7c44ef.png">

Reply:
lg2m